packages/web-components/fast-element/src/directives/node-observation.ts

```diff
@@ -2,22 +2,54 @@ import { Accessor, Observable } from "../observation/observable";
 import { emptyArray } from "../interfaces";
 import { Behavior } from "./behavior";
 
+/**
+ * Options for configuring node observation behavior.
+ * @public
+ */
 export interface NodeBehaviorBehaviorOptions<T = any> {
+    /**
+     * The property to assign the observed nodes to.
+     */
     property: T;
 }
 
+/**
+ * A base class for node observation.
+ * @internal
+ */
 export abstract class NodeObservationBehavior<T extends NodeBehaviorBehaviorOptions>
     implements Behavior {
     private source: any = null;
     private shouldUpdate!: boolean;
 
+    /**
+     *
```

Comment:
"Creates an instance of NodeObservationBehavior"?

Reply:
On it.